sphinx_gallery/gen_gallery.py

```diff
@@ -422,6 +422,9 @@ def generate_gallery_rst(app):
     Start the Sphinx-Gallery configuration and recursively scan the examples
     directories in order to populate the examples gallery
     """
+    if app.builder.name != 'html':
+        raise ConfigError('Builder must be `html` for sphinx-gallery')
```

Comment:
I think technically we do support LaTeX as well. Maybe better to make this a blocklist (containing just `dirhtml` for now) rather than an allowlist?

Reply:
Yes, probably a good idea. 

However, I have not checked other builders than `html` and `dirhtml` yet.

Reply:
I think that's okay. We can just add the broken ones as we come along them. We can take the easy way out and presume innocent until proven guilty here :)

Reply:
Then it might be a good idea if I add this also in the docu. Could you please point me to the location where you would put this info.

Reply:
Maybe a `.. note::` at the bottom of this main/topmost section:

https://sphinx-gallery.github.io/stable/getting_started.html#creating-a-basic-gallery

Saying that the HTML and LaTeX builders are the only ones that have been tested/targeted so far? The corresponding place I think would be here:

https://github.com/sphinx-gallery/sphinx-gallery/blame/master/doc/getting_started.rst#L26